src/main/resources/aws/virtual-users.yaml

```diff
@@ -39,6 +39,7 @@ Resources:
           Ebs:
             VolumeSize: 30
             VolumeType: gp2
+            Encrypted: true
```

Comment:
@mzyromski-atlassian FYI this was missing a changelog entry

Reply:
Thanks for the info. I'll include it in the changelog after this gets released.